platform/index.d.ts

```diff
@@ -1,10 +1,9 @@
-// Type definitions for Platform 1.0.0
+// Type definitions for Platform 1.3
 // Project: https://github.com/bestiejs/platform.js
 // Definitions by: Jake Hickman <https://github.com/JakeH/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-
-interface PlatformStatic {
+declare interface Platform {
```

Comment:
What do you mean. Here is a working example:
![img-2017-03-21-10-31-26](https://cloud.githubusercontent.com/assets/803674/24136802/964e6e76-0e21-11e7-8e3d-9faac63664f3.png)